br-ext/board/qemu/post-build.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
```

Comment:
I think a copyright notice would be nice (despite other shell scripts don't have).

Reply:
Where can I find an example of such a copyright notice?

Reply:
```bash
#! /bin/bash
# SPDX-License-Identifier: BSD-2-Clause
#
# Copyright (c) 2020, ...
```